### PR TITLE
Handling legacy group requests

### PR DESCRIPTION
**Pull request by michaltrzesimiech**
Closes https://github.com/rtcharity/eahub.org/issues/604

### CODE REVIEW

File: eahub/config/urls.py [hunk 1]
```diff
@@ -36,8 +36,9 @@
     path('groups/get-a-website/', views.LegacyRedirectView.as_view(url='https://github.com/rtcharity/lean-site-template')),
     path('groups/local-group-support-overview-lean-cea-and-eaf/', views.LegacyRedirectView.as_view(url='https://forum.effectivealtruism.org/posts/Cvi7hnTYMk5qutkDg/local-effective-altruism-network-s-new-focus-for-2019')),
     path('groups/resources/giving-games/', views.LegacyRedirectView.as_view(url='https://docs.google.com/document/d/1g5G0PvYFs7cAbAZ8ANI_wnsv7DjtBmq4_SQcg44-HDA/edit')),
+    path('groups/<slug:slug>/', views.LegacyRedirectView.as_view(pattern_name='groups')),
```
Comment (taymonbeal) [inline eahub/config/urls.py]: This doesn't look right; I'd expect it to be `pattern_name="group"`. Have you tested it locally?

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: You're right. 

> Have you tested it locally?

I wasn't able to set up the local env. Raised an issue here: https://github.com/rtcharity/eahub.org/issues/709

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: I tested it now and it looks like we want `groups`. `localhost:8000/group` returns 404.

Comment (taymonbeal) [reply eahub/config/urls.py]: The pattern you want to be pointing at is https://github.com/rtcharity/eahub.org/blob/df156c575b7ebfb74c1537ca82f228f9c04b8ac5/eahub/localgroups/urls.py#L10. `/group` is indeed not a URL that we have, but I don't see what that has to do with anything; it accepts a pattern name, not a URL path. Did you not get a 500 error with `pattern_name='groups'`?

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: Sorry, I got confused. Obviously, you're right. I tested it and it works as it' supposed.

**Pull request merged**